modules/openapi-generator/src/main/resources/Java/build.gradle.mustache

```diff
@@ -125,7 +125,7 @@ ext {
     {{/useBeanValidation}}
     jersey_version = "1.19.4"
     jodatime_version = "2.9.9"
-    junit_version = "4.13.2"
+    junit_version = "5.10.2"
```

Comment:
can you please update pom.xml template as well?

Reply:
I think I've done that already: https://github.com/OpenAPITools/openapi-generator/pull/18222/files/5d06e69dbe323be0cfd3c3df5c68b13b0ebd05fb#diff-545248c48d06e93aa2fe174a53a80ed654eb113f124f7f2f391ed60ea7bb2e91L365-R380